go-controller/pkg/node/node_ip_handler_linux.go

```diff
@@ -62,8 +62,17 @@ func (c *addressManager) delAddr(ip net.IP) bool {
 }
 
 func (c *addressManager) Run(stopChan <-chan struct{}) {
+	errorCallback := func(err error) {
+		klog.Errorf("Failed during AddrSubscribe callback: %v", err)
+	}
+	addrSubscribeOptions := netlink.AddrSubscribeOptions{
```

Comment:
@flavio-fernandes All you need here is `ErrorCallback` since you're just setting everything else to the default values.

Reply:
Make sense. Done.